chunky/src/java/se/llbit/chunky/chunk/biome/BiomeData2d.java

```diff
@@ -34,7 +35,7 @@ public void clear() {
   }
 
   public static void loadBiomeDataByteArray(Tag chunkData, BiomeData2d biomeData, BiomePalette biomePalette) {
```

Comment:
Should this be moved to the world? Seems to be world format / chunk data dependant.